presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -716,6 +716,56 @@ public void testSchemaAuthorization()
         assertUpdate(admin, "DROP ROLE admin");
     }
 
+    @Test
+    public void testShowCreateSchema()
```

Comment:
We need also add simple test to `io.prestosql.testing.AbstractTestIntegrationSmokeTest`. That file is used to test connector implementations, you change connector SPI so we need tests for other connectors. It can be very simple, without any access control. Probably for most of the connectors it will be skipped.

Reply:
I'm not sure what I would test.

It's kinda like `TestHiveIntegrationSmokeTest.testSchemaOperations`, which _only_ exists in `TestHiveIntegrationSmokeTest`, because it doesn't make sense for many other connectors, and there's no equivalent for this test in the other connector tests.

Do you have a suggestions about what I can test generally?

Reply:
I was thinking to add simple `SHOW CREATE SCHEMA tpch`. If connector does not support this, then we could disable that test for this connector, but it will be known that there is a feature that given connector does not support.

Reply:
I see. Makes sense.
Should probably pull `TestHiveIntegrationSmokeTest.testSchemaOperations` up to `AbstractTestIntegrationSmokeTest` as well then.

Reply:
Makes sense